devel/googletest/Portfile

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+
+github.setup        google googletest 1.10.0 release-
+revision            0
+categories          devel
+
+license             BSD
+maintainers         nomaintainer
+platforms           darwin
+
+description         Google's C++ test framework
+
+long_description    googletest is a testing framework developed by the \
+                    Testing Technology team with Google's specific \
+                    requirements and constraints in mind.
+
+github.tarball_from archive
+
+checksums           rmd160  f2f7f96adf1ab8aae3f6204fdba3ff150fd4396b \
+                    sha256  9dc9157a9a1551ec7a7e43daea9a694a0bb5fb8bec81235d8a1e6ef64c716dcb \
+                    size    904349
```

Comment:
please drop this commit, the port is already present as `devel/gtest`

Reply:
Oops, thanks, done.